### ISSUE TITLE
t/utility/xl_parse_date.t warns: the TZ Date::Manip config variable is deprecated

### ISSUE THREAD

**Issue by ppisar** [jmcnamara/excel-writer-xlsx#307]
Executing t/utility/xl_parse_date.t test with Date-Manip-6.95 warns:

~~~~
$ perl -Ilib t/utility/xl_parse_date.t 
1..4
WARNING: the TZ Date::Manip config variable is deprecated
         and will be removed in version 7.00.  Please use
         the SetDate or ForceDate config variables instead.
 at /usr/share/perl5/vendor_perl/Date/Manip/Obj.pm line 286.
ok 1 -          Utility: xl_parse_date( 2/1/2002 )
ok 2 -          Utility: xl_parse_date( 11/7/97 )
ok 3 -          Utility: xl_parse_date( Friday 11 July 1997 )
ok 4 -          Utility: xl_parse_date( 10:12 AM Friday 11 July 1997 )
~~~~

This can be reduced to:

~~~~
$ perl -Ilib -e 'use Excel::Writer::XLSX::Utility; Excel::Writer::XLSX::Utility::xl_parse_date_init("TZ=GMT")'
WARNING: the TZ Date::Manip config variable is deprecated
         and will be removed in version 7.00.  Please use
         the SetDate or ForceDate config variables instead.
 at /usr/share/perl5/vendor_perl/Date/Manip/Obj.pm line 286.
~~~~

The problem is in arguments passed to DateTime::Manip::Date_Init().

When fixing this, be ware that the TZ variable is not only passed from the test but also mentioned in Excel::Writer::XLSX::Utility documentation.

**Comment by jmcnamara** [comment]
Fixed on main. If it looks okay then I will roll it, and the MANIFEST fix, out in a new release.

**Comment by ppisar** [comment]
It looks good.

**Comment by jmcnamara** [comment]
Fixed in version 1.14. Thanks.
